### PR TITLE
Add action types

### PR DESCRIPTION
**Pull request by LabhanshAgrawal**
Adding types for actions, to be used in reducer.

### CODE REVIEW

File: lib/constants/ui.ts [hunk 1]
```diff
@@ -22,3 +22,104 @@ export const UI_ENTER_FULLSCREEN = 'UI_ENTER_FULLSCREEN';
...
+}
+interface UIContextmenuOpenAction {
+  type: typeof UI_CONTEXTMENU_OPEN;
+}
+interface UICommandExecAction {
+  type: typeof UI_COMMAND_EXEC;
+}
+
+export type UIActions =
+  | UIFontSizeSetAction
```
Comment (ppot) [inline lib/constants/ui.ts]: Shouldn't there be no `|` before `UIFontSizeSetAction ` ?

Comment (LabhanshAgrawal) [reply lib/constants/ui.ts]: It's actually inserted by Prettier, allows for easier addition/removal/rearranging of values. Similar to extra comma after the last value in a list.

Comment (ppot) [reply lib/constants/ui.ts]: Oh okay

Comment (LabhanshAgrawal) [reply lib/constants/ui.ts]: It's just syntax sugar, doesn't impact code logic/execution

**Comment by LabhanshAgrawal** [comment]
@ppot Is there anything else blocking this?

**Pull request merged**